.travis.yml

```diff
@@ -3,8 +3,12 @@ python:
     - "2.7"
 install:
     - pip install requests junit_xml beaker-client mock
-    - pip install pylint pycodestyle
+    - pip install pylint pycodestyle flake8
+before_script:
+    - flake8 --show-source .
+    - pycodestyle .
```

Comment:
Quoting the flake8's doc:
```
Flake8 is a wrapper around PyFlakes, pycodestyle, and Ned's McCabe script.
Flake8 runs all tools by launching the single 'flake8' script, but ignores pycodestyle and PyFlakes extended options and just uses defaults.
```
So I think that you should remove `pycodestyle .` item.

Reply:
Ah, good catch, @agustinhenze. I'll fix that now.